src/api/hooks/useGetValidators.ts

```diff
@@ -64,9 +64,10 @@ function useGetValidatorsRawData(network: NetworkName) {
 }
 
 export function useGetValidators(network?: NetworkName) {
+  const [state] = useGlobalState();
   const {activeValidators} = useGetValidatorSet();
   const {validatorsRawData} = useGetValidatorsRawData(
-    network ?? Network.MAINNET,
+    network ?? state.network_name,
```

Comment:
don't default on mainnet, default on the current network name selection

Reply:
can we also test the behavior for when network changes?